src/components/testimonial/Testimonial.jsx

```diff
@@ -0,0 +1,29 @@
+import "./Testimonial.css"
+import PlayButton from "../playbutton/PlayButton.jsx";
+import {ArrowRight} from "@phosphor-icons/react"
+
+function Testimonial({img, title, text, altText, onclick, url}) {
+  console.log(url)
+  return (
+    <article className="testimonial-card">
+      <div onClick={() => onclick(url)} className="image-container">

```

Comment:
Het is gebruiksvriendelijker om de pagina te laten bepalen wat er op de onClick moet gebeuren. Dus je kunt de `handleClick` functie direct aan `onClick` meegeven zonder extra toeters en bellen!